docs/external-node.md

```diff
@@ -266,13 +266,24 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
    ```
 
 2. The `antrea-agent.conf` file specifies agent configuration parameters. Copy
```

Comment:
Not relevant to this PR, but a reminder we should document downloading the releases binaries, once we have them.

Reply:
cc @wenyingd